src/mqtt.py

```diff
@@ -57,6 +57,7 @@ def connect(self):
 
     def disconnect(self):
         self._client.disconnect()
+        self._queue.put(None)
```

Comment:
If I don't do that I stay stuck in `data = client._queue.get()`. Get will block until a MQTT event is received. I'll make another test tomorrow.